src/service/mercury/index.ts

```diff
@@ -245,7 +241,7 @@ export class MercuryClient {
       };
 
       const { transferFromRes, transferToRes, mintRes } =
-        await this.renewAndRetry(subscribe, network);
+        await this.renewAndRetry(subscribe, network, 5);
```

Comment:
NIT: maybe we should set 5 as a const like `DEFAULT_RETRY_AMOUNT` to keep these in sync and give a bit of an explanation as to what the number means at a glance

Reply:
yeah good call, added in 73d2f5216802f61c2784c80acb665ed24e6f813e